src/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
I think you can't use margin for BEM block

Reply:
Maybe, but I am not sure